building/build-helpers/helper-go/.gitignore

```diff
@@ -1 +1 @@
-go
+go*
```

Comment:
This should be more specific so as not to match `go1.4-bootstrap-20170531.tar.xz` and `go1.8.3.src.tar.xz` which are already in the repository as symlinks:

```
/go*/
```

(Leading slash: only match immediate children of this directory `helper-go`. Trailing slash: only match those that are themselves directories.)

Reply:
I'm addressing this in a separate commit for the sake of moving things along.

Reply:
I'm addressing this in a separate commit for the sake of moving things along.